src/Uno.UWP/Extensions/PermissionsHelper.cs

```diff
@@ -36,7 +36,9 @@ internal static void Initialize(FuncAsync<string, bool> getter, FuncAsync<string
 		public static bool IsDeclaredInManifest(string permission)
 		{
 			var context = Application.Context;
+#pragma warning disable CS0618 // Type or member is obsolete
```

Comment:
Should we open an issue for all those deprecated members?

Reply:
Most of the obsolete members continue to work properly, so we'll eventually change those if the APIs end up being removed (which is unlikely to happen, given android's history).